x/examples/website-wrapper-app/doctor

```diff
@@ -0,0 +1,206 @@
+#!/bin/bash
+
+# Copyright 2025 The Outline Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+YELLOW='\033[1;33m'
+BLUE='\033[0;34m'
+NC='\033[0m'
+
+CHECK_MARK="✅"
+CROSS_MARK="❌"
+WARNING_MARK="⚠️"
+INFORMATION_MARK="ℹ️"
+
+URL_NODE="https://nodejs.org/en/download/"
+URL_GO="https://go.dev/doc/install"
+MIN_GO_VERSION="1.21"
+URL_XCODE="https://developer.apple.com/xcode/"
+HINT_XCODE_CLI="Try running 'xcode-select --install' in your terminal, or install Xcode from the App Store."
+URL_COCOAPODS="https://cocoapods.org/"
+HINT_COCOAPODS="Try running 'sudo gem install cocoapods'"
```

Comment:
Oops, it seems another dependency is introduced: `gem` (or `Ruby`). But maybe we don't need to specifically mention it, the developer should be able to figure it out. What do you think?